src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1267,7 +1268,7 @@ public function testOverwritingDefaultInputDefinitionOverwritesDefaultValues()
         $inputDefinition = $application->getDefinition();
 
         // check whether the default arguments and options are not returned any more
-        $this->assertFalse($inputDefinition->hasArgument('command'));
+        $this->assertTrue($inputDefinition->hasArgument('command'));
```

Comment:
This looks like you've changed existing behavior. Are you sure this won't break existing applications?

Reply:
I am re-using `ApplicationTest` but changed its `getDefaultInputDefinition` to also contain a command which was required by another test. I can create a new `ApplicationTest2` if that is problematic, but this should not break existing applications.

Reply:
Created another class to avoid confusion.